src/components/entry/EntryDetails.js

```diff
@@ -1,19 +1,253 @@
+import { Wikipedia } from '@styled-icons/boxicons-logos'
+//import { Calendar } from '@styled-icons/boxicons-regular'
+import { Flag, Map, Star } from '@styled-icons/boxicons-solid'
+import React, { useEffect, useState } from 'react'
 import { useRouteMatch } from 'react-router-dom'
+import styled from 'styled-components'
 
-const EntryDetails = () => {
+import { getLocationById, getTypeById } from '../../utils/api'
+import Button from '../ui/Button'
+import { theme } from '../ui/GlobalStyle'
+import IconButton from '../ui/IconButton'
+import { Tag } from '../ui/Tag'
+import EatTheWeedsLogo from './EatTheWeeds.svg'
+import ForagingTexasLogo from './ForagingTexas.png'
+import FruitipediaLogo from './Fruitipedia.png'
+import UrbanMushroomsLogo from './UrbanMushrooms.png'
+import USDALogo from './USDA.svg'
+
+const ACCESS_TYPE = {
```

Comment:
i think it'd be useful to extract this into its own utils file potentially, since there are other enums also associated with entry details (Fruiting Status, Quality Rating, and Yield Rating) that can be used elsewhere in the future. 

the list of logos/urls could also potentially use its own utils file like ymca did here: https://github.com/hack4impact-uiuc/ymca/blob/master/client/src/utils/determineStockPhoto.js